packages/pulpcore/python-importlib-metadata/python-importlib-metadata.spec

```diff
@@ -0,0 +1,48 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name importlib-metadata
+
+Name:           python-%{pypi_name}
+Version:        1.4.0
+Release:        1%{?dist}
+Summary:        Read metadata from Python packages
+
+License:        Apache Software License
+URL:            http://importlib-metadata.readthedocs.io/
+Source0:        https://files.pythonhosted.org/packages/source/i/%{pypi_name}/importlib_metadata-%{version}.tar.gz
```

Comment:
What a weird combination of dashes and underscores.

Reply:
:woman_shrugging: